be/src/Project/Web/Buyer/Buyer.Web/Shared/Services/Clients/ClientsService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Buyer.Web.Shared.Definitions.CompletionDate;
+using Buyer.Web.Shared.Repositories.Clients;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Buyer.Web.Shared.Services.Clients
+{
+    public class ClientsService : IClientsService
+    {
+        private readonly IClientsRepository _clientsRepository;
+
+        public ClientsService(IClientsRepository clientsRepository)
+        { 
+            _clientsRepository = clientsRepository;
+        }
+
+        public async Task<bool> IsEltapTransportEnableAsync(string token, string language, Guid? sellerId)
```

Comment:
**suggestion:** it's bad design to have Eltap specific code here. We need to refactor it completely